<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find</class>
 <widget class="QWidget" name="Find">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COMP_hack Capture Grep - Find</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="binaryButton">
       <property name="toolTip">
        <string>Search for binary data in a packet.</string>
       </property>
       <property name="text">
        <string>Binary</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="textButton">
       <property name="toolTip">
        <string>Search for text in a packet.</string>
       </property>
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="commandButton">
       <property name="toolTip">
        <string>Search for packets of a specific command.</string>
       </property>
       <property name="text">
        <string>Command</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="findEdit">
     <property name="toolTip">
      <string>What to search for.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="encodingLabel">
       <property name="text">
        <string>Encoding:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="encoding">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Encoding of the string being searched for.</string>
       </property>
       <item>
        <property name="text">
         <string>CP1252 (Windows  - English)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CP932 (Windows - Japanese)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTF-8 (Unicode)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout2">
     <item>
      <widget class="QPushButton" name="findButton">
       <property name="toolTip">
        <string>Search for the above term.</string>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="toolTip">
        <string>Close the search dialog.</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="searchLabel">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="findList">
     <property name="toolTip">
      <string>Results of the search.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findEdit</tabstop>
  <tabstop>encoding</tabstop>
  <tabstop>findButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>findList</tabstop>
  <tabstop>binaryButton</tabstop>
  <tabstop>textButton</tabstop>
  <tabstop>commandButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
